tests/ha/test_ha_pod_restart.py

```diff
@@ -2878,7 +2878,7 @@ def test_continuous_reads_during_pod_restart_ros(self):
         self.s3_clean.update(users)
         resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
                                                     log_prefix=self.test_prefix,
-                                                    skipcleanup=True, nclients=5, nsamples=5)
+                                                    skipcleanup=True, nclients=20, nsamples=20)
```

Comment:
Why workload is increased?

Reply:
I ran the test 3 times, and observed with old sample value, the read operation was getting completed even before re-start of cluster gets completed (i.e step 7 and Step 8). So just to get enough time till cluster completion gets completed, increased the sample value. This test also failed in LC environment. So not able to complete end to end test.